### PR TITLE
Use a quick parsing configuration for symbol solver unit tests

### PR DESCRIPTION
**Pull request by matozoid**
Disabling a bunch of parser features to get the symbol solver to run quicker.

### CODE REVIEW

**Comment by matozoid** [comment]
Looks like this takes 25% off the build time. Still a lot, but better.

File: javaparser-symbol-solver-testing/src/test/java/com/github/javaparser/symbolsolver/Issue113.java [hunk 1]
```diff
@@ -25,7 +26,7 @@
 
     @Before
     public void setup() {
-        typeSolver = new CombinedTypeSolver(new ReflectionTypeSolver(), new JavaParserTypeSolver(adaptPath("src/test/resources/issue113")));
+        typeSolver = new CombinedTypeSolver(new ReflectionTypeSolver(), new JavaParserTypeSolver(adaptPath("src/test/resources/issue113"), new LeanParserConfiguration()));
```
Comment (ftomassetti) [inline javaparser-symbol-solver-testing/src/test/java/com/github/javaparser/symbolsolver/Issue113.java:29]: Nice name!

File: javaparser-symbol-solver-testing/src/test/java/com/github/javaparser/symbolsolver/Issue128.java [hunk 2]
```diff
@@ -30,7 +31,7 @@
     @Before
     public void setup() throws IOException {
         Path srcDir = adaptPath("src/test/resources/issue128");
-        typeSolver = new CombinedTypeSolver(new ReflectionTypeSolver(), new JavaParserTypeSolver(srcDir));
+        typeSolver = new CombinedTypeSolver(new ReflectionTypeSolver(), new JavaParserTypeSolver(srcDir, new LeanParserConfiguration()));
```
Comment (ftomassetti) [inline javaparser-symbol-solver-testing/src/test/java/com/github/javaparser/symbolsolver/Issue128.java:34]: Should we use singleton for `LeanParserConfiguration`? Or for that we would need to make it immutable?

File: javaparser-symbol-solver-testing/src/test/java/com/github/javaparser/symbolsolver/Issue1485.java [hunk 3]
```diff
@@ -26,7 +27,7 @@ public void issue1485withoutSpecifyingJARs() throws IOException {
 
         CombinedTypeSolver typeSolver = new CombinedTypeSolver();
```
Comment (ftomassetti) [inline javaparser-symbol-solver-testing/src/test/java/com/github/javaparser/symbolsolver/Issue1485.java:28]: Perhaps we could have an helper method in an ancestor test class that just create this combined type solver with reflection + the Jptypesolver

**Review by ftomassetti — APPROVED** [review]
Looks good!

File: javaparser-symbol-solver-testing/src/test/java/com/github/javaparser/symbolsolver/Issue128.java (hunk 2 above)
Comment (matozoid) [reply javaparser-symbol-solver-testing/src/test/java/com/github/javaparser/symbolsolver/Issue128.java:34]: That would be extra code, and now we're wasting maybe a few bytes in a testrun that takes gigabytes :-)

File: javaparser-symbol-solver-testing/src/test/java/com/github/javaparser/symbolsolver/Issue1485.java (hunk 3 above)
Comment (matozoid) [reply javaparser-symbol-solver-testing/src/test/java/com/github/javaparser/symbolsolver/Issue1485.java:28]: I'm wary of abstraction in test code, so I'll leave that to you.

File: javaparser-symbol-solver-testing/src/test/java/com/github/javaparser/symbolsolver/Issue113.java (hunk 1 above)
Comment (matozoid) [reply javaparser-symbol-solver-testing/src/test/java/com/github/javaparser/symbolsolver/Issue113.java:29]: Thanks! We could rename ParserConfiguration to FatParserConfiguration :-)

**Pull request merged**